### PR TITLE
XIVY-3162 light-hearted appraoch to raise-leading edge repos

### PR DESCRIPTION
**Pull request by ivy-rew**

### CODE REVIEW

**Comment by ivy-rew** [comment]
github's response isn't nice yet, but still this already solves the 'raise-project-version' problem for our leading-edge repos

**Review by alexsuter — APPROVED** [review]

**Comment by alexsuter** [comment]
> I'll try to add the Portal too rocket

Better not - because this is on bitbucket. We have no bitbucket integration anymore. I think portal needs to migrate as core component also in this space.

**Comment by ivy-rew** [comment]
> > I'll try to add the Portal too rocket
> 
> Better not - because this is on bitbucket. We have no bitbucket integration anymore. I think portal needs to migrate as core component also in this space.

good point, yes I think we should make them an offer to help with the migration to github ... and they will in return gain auto-updated projects.

File: build/raise-ivy-projects-version/raise.sh [hunk 1]
```diff
@@ -82,6 +82,9 @@ raiseProjects() {
 updateProjectRepos() {
   projectRepos=(
     "git@github.com:axonivy/core.git"
+    "git@github.com:axonivy-market/demo-projects.git"
+    "git@github.com:axonivy-market/doc-factory.git"
+    "git@github.com:axonivy-market/msgraph-connector.git"
   )
```
Comment (ivy-rew) [inline build/raise-ivy-projects-version/raise.sh:88]: @weissreto the raise-ivy-project build is now able to update all our leading-edge repos at once :rocket:

**Pull request merged**